README.md

```diff
@@ -53,124 +50,38 @@ import tidy3d.web as web
 web.configure("XXX")
 ```
 
-If those commands did not work, advanced installation instructions are below, which should help solve the issue.
+**Advanced installation instructions for all platforms is available in the [documentation installation guides](https://docs.flexcompute.com/projects/tidy3d/en/latest/install.html).**
 
-### Advanced Installation Instructions
+### Authentication Verification
 
-Some users might require more a specialized installation, which we cover below.
-
-#### Using pip (recommended)
-
-The easiest way to install the tidy3d python interface is through [pip](https://pypi.org/project/tidy3d/).
-
-```
-pip install tidy3d
-```
-
-This will install the latest stable version.
-
-To get a specific version `x.y.z`, including the "pre-release" versions, you may specify the version in the command as:
-
-```
-pip install tidy3d==x.y.z
-```
-
-### Installing from source
-
-Alternatively, for development purposes, eg. developing your own features, you may download and install the package from source as:
-
-```
-git clone https://github.com/flexcompute/tidy3d.git
-cd tidy3d
-pip install -e .
-```
-
-### Configuring and authentication
-
-With the front end installed, it must now be configured with your account information, which is done via an "API key".
-
-You can find your API key in the [web interface](http://tidy3d.simulation.cloud). After signing in and navigating to the account page by clicking the "Account Center" icon on the left-hand side. Then, click on the "API key" tab on the right hand side of the menu and copy your API key.
-
-Note: We refer to your API specific API key value as `XXX` below.
-
-To link your API key with Tidy3D, you may use one of three following options:
-
-#### Command line (recommended)
-
-The easiest way is through the command line via the `tidy3d configure` command. Run:
-
-```
-tidy3d configure
-```
-
-and then enter your API key `XXX` when prompted.
-
-Note that Windows users must run the following instead (ideally in an anaconda prompt):
-
-```
-pip install pipx
-pipx run tidy3d configure
-```
-
-You can also specify your API key directly as an option to this command using the `api-key` argument, for example:
-
-```
-tidy3d configure --apikey=XXX
-```
-
-#### Manually
-
-Alternatively, you can manually set up the config file where Tidy3D looks for the API key. The API key must be in a file called `.tidy3d/config` located in your home directory, containing the following
+To test the authentication, you may try importing the web interface via.
```

Comment:
I'm not sure this triggers the authentication. Could be wrong. There's a `tidy3d.web.test()` function that will do this.

https://github.com/flexcompute/tidy3d/blob/develop/tidy3d/web/api/webapi.py#L818-L833